CHANGELOG.md

```diff
@@ -1203,7 +1203,7 @@ code.
 - Constants `AVIF_VERSION`, `AVIF_VERSION_MAJOR`, `AVIF_VERSION_MINOR`, `AVIF_VERSION_PATCH`
 - `avifVersion()` function
 
-[Unreleased]: https://github.com/AOMediaCodec/libavif/compare/v1.1.1...HEAD
+[1.2.0]: https://github.com/AOMediaCodec/libavif/compare/v1.1.1...v1.2.0
```

Comment:
Similarly, this should be a new line after the `[Unreleased]:` line, which should be updated to:

```
[Unreleased]: https://github.com/AOMediaCodec/libavif/compare/v1.2.0...HEAD
```

Reply:
Same.